storage/txcache/score.go

```diff
@@ -38,25 +38,21 @@ func (computer *defaultScoreComputer) computeScore(scoreParams senderScoreParams
 
 // TODO (optimization): switch to integer operations (as opposed to float operations).
 func (computer *defaultScoreComputer) computeRawScore(params senderScoreParams) float64 {
-	allParamsDefined := params.fee > 0 && params.gas > 0 && params.size > 0 && params.count > 0
+	allParamsDefined := params.feeScore > 0 && params.gas > 0 && params.count > 0
 	if !allParamsDefined {
 		return 0
 	}
 
-	PPUMin := float64(computer.minGasPrice)
-	PPUAvg := float64(params.fee) / float64(params.gas)
-	PPUScore := math.Pow(PPUAvg/PPUMin, 3)
-
-	countPow2 := float64(params.count) * float64(params.count)
-	countScore := math.Log(countPow2+1) + 1
+	ppuMin := computer.txFeeHelper.minPricePerUnit()
+	ppuAvg := params.feeScore / (params.gas >> computer.txFeeHelper.gasLimitShift())
```

Comment:
is there any advantage of shifting params.gas to the right instead of shifting the params.feescore to the left? The params.gas is checked  to be >0 but could be zero when shifting to the right?

Reply:
gas params.gas here should be the not yet normalized gas accumulated for one account from all it's txs still in mempool
the normalization shift is computed in relation to minimum gas required, so it should never lead to 0 here.
added all the same a condition to avoid this.

Reply:
I would say it would have been better to shift feescore to the left first (to have more accurate resolution), but since the orders of magnitude for gasPrice and gassLimit are configurable and the formula adjusts, there will be more complex conditions to ensure there is no overflow.

Reply:
ok